### PR TITLE
WEBEU2 - Responsive Web Design I - Josemaria Nriagu 🇳🇬

### PR DESCRIPTION
**Pull request by josenriagu**
My First PR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,16 +1,96 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>Great Idea - Responsive I</title>
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (maximesalomon) [inline index.html]: 👍

File: index.html [hunk 2]
```diff
@@ -1,16 +1,96 @@
...
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+      <div class="headerimage">
+        <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+      </div>
+    </header>
+    <!-- end header section -->
+    <!-- start main body section -->
+    <main>
```
Comment (maximesalomon) [inline index.html]: Good use of semantic HTML `<main>` 👍

File: index.html [hunk 3]
```diff
@@ -1,16 +1,96 @@
...
+      <!-- top content -->
+      <div class="top-content">
+        <div class="top-text">
+          <h1>Innovation<br>On<br>Demand</h1>
```
Comment (maximesalomon) [inline index.html]: I won't recommend using `<br>` like this

File: index.html [hunk 4]
```diff
@@ -1,16 +1,96 @@
...
+          <button type="button">Get Started</button>
+        </div>
+        <div class="top-image"><img src="img/header-img.png" alt="Image of a code snippet."></div>
+      </div>
+      <!-- middle content -->
```
Comment (maximesalomon) [inline index.html]: Good job using comments

File: index.html [hunk 5]
```diff
@@ -1,16 +1,96 @@
...
+          <h3>Vision</h3>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse
+            eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </section>
+      </div>
+      <!-- bottom content -->
+      <div class="bottom-content">
+        <h3>Contact</h3>
+        <p>123 Way 456 Street<br>Somewhere, USA</p>
```
Comment (maximesalomon) [inline index.html]: You might want to use `<address>` here - https://www.w3schools.com/tags/tag_address.asp

File: services.html [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+  <div class="srvcontainer">
+    <div class="srvheader">
```
Comment (maximesalomon) [inline services.html]: Where is the `<header>` tag?

File: services.html [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+      <div class="headernav">
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </div>
```
Comment (maximesalomon) [inline services.html]: Where is the `</nav>` tag?

File: services.html [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+      <div class="headerimage">
+        <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+      </div>
+    </div>
+    <div class="section1">
+      <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+    </div>
+    <div class="section2">
```
Comment (maximesalomon) [inline services.html]: Where is the `<main>` tag?

File: services.html [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+      <div class="big patty">
+        <h3>UX / UI</h3>
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem
+          neque, sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.</p>
+        <button type="button">Learn More</button>
+      </div>
+    </div>
+    <div class="section3b">
```
Comment (maximesalomon) [inline services.html]: Use `<section>` and not `<div>`

File: services.html [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+          <li>Awesome things</li>
+          <li>Amazing things</li>
+          <li>Cool things</li>
+          <li>Great things</li>
+        </ul>
+      </div>
+      <div class="two4"><img class="services-info-img" src="img/services-info.png"
+          alt="one of our employees hard at work"></div>
+    </div>
+    <div class="srvfooter">
```
Comment (maximesalomon) [inline services.html]: Where is the `<footer>` tag?

File: css/index.css [hunk 11]
```diff
@@ -1,3 +1,495 @@
...
+}
+
+/* set container styles */
+
+.container {
+    max-width: 900px;
+    margin: 0 auto;
+}
+
+/* set header styles and media queries */
```
Comment (maximesalomon) [inline css/index.css]: Good use of comments 👍

File: css/index.css [hunk 12]
```diff
@@ -1,3 +1,495 @@
...
+.bottom-content {
+    width: 30%;
+    margin-top: 1.9rem;
+}
+
+.bottom-content p {
+    padding-bottom: 0.63rem;
+}
+
+@media(max-width: 800px) {
```
Comment (maximesalomon) [inline css/index.css]: Capture d’écran 2019-04-29 à 18 49 10
check the margin

**Comment by josenriagu** [comment]
I designed services page before the peer code review last week. During this
review, I learnt some great lessons from my peer. I will try to fix the
semantic issues as soon as possible. That's a whole new redesign.

Thanks a lot for your support.

Regards,

Josemaria Nriagu
🇳🇬

**Pull request merged**